src/psyclone/tests/psyir/transformations/value_range_check_test.py

```diff
@@ -35,105 +35,106 @@
 # Modified by: R. W. Ford, S. Siso and N. Nobre, STFC Daresbury Lab
 # -----------------------------------------------------------------------------
 
-''' Module containing tests for NanTestTrans and NanTestNode
+''' Module containing tests for ValueRangeCheck and ValueRangeCheckNode
 '''
 
-from __future__ import absolute_import
-
 import pytest
 
+
 from psyclone.errors import InternalError
-from psyclone.psyir.nodes import colored, Node, NanTestNode, Schedule
-from psyclone.psyir.transformations import (NanTestTrans,
+from psyclone.psyir.nodes import colored, Node, ValueRangeCheckNode, Schedule
+from psyclone.psyir.transformations import (ValueRangeCheck,
                                             TransformationError)
 from psyclone.tests.utilities import get_invoke
 from psyclone.transformations import OMPParallelLoopTrans
 
 
-# --------------------------------------------------------------------------- #
+# ---------------------------------------------------------------------------
 def test_extract_trans():
```

Comment:
s/extract/value_range_check/ and again for the 'malformed' test at L63.

Reply:
Done.